homeassistant/components/lyric/climate.py

```diff
@@ -191,8 +196,14 @@ def __init__(
     def supported_features(self) -> ClimateEntityFeature:
         """Return the list of supported features."""
         if self.device.changeableValues.thermostatSetpointStatus:
-            return SUPPORT_FLAGS_LCC
-        return SUPPORT_FLAGS_TCC
+            supported_flags = SUPPORT_FLAGS_LCC
+        else:
+            supported_flags = SUPPORT_FLAGS_TCC
+
+        if self._attr_fan_modes:
+            supported_flags = supported_flags | ClimateEntityFeature.FAN_MODE
+
+        return supported_flags
```

Comment:
Instead of a property, can we set `self._attr_supported_features` at `__init__()`?

Reply:
Yes - I moved it to `__init__`.